From 16465afc252c429111ff6553556ac2f018a8c281 Mon Sep 17 00:00:00 2001
From: Adrien Plazas <kekun.plazas@laposte.net>
Date: Wed, 12 Jan 2022 15:51:43 +0100
Subject: [PATCH] progress-button: Ellipsize the label

This ensures the buttons don't grow so wide they make the window pop out
of the screen.
---
 src/gs-progress-button.ui | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/gs-progress-button.ui b/src/gs-progress-button.ui
index daa28d28f..3407c332a 100644
--- a/src/gs-progress-button.ui
+++ b/src/gs-progress-button.ui
@@ -9,6 +9,7 @@
         <property name="visible">True</property>
         <child>
           <object class="GtkLabel" id="label">
+            <property name="ellipsize">end</property>
             <property name="label" bind-source="GsProgressButton" bind-property="label" bind-flags="sync-create"/>
             <property name="use-underline" bind-source="GsProgressButton" bind-property="use-underline" bind-flags="sync-create"/>
             <property name="visible">True</property>
-- 
2.30.2

